### PR TITLE
Add several companies and several links

### PR DESCRIPTION
**Pull request by lucamug**
Add several companies and several links to posts, talks, and videos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,7 +234,7 @@ Inspired by Doomspork's [Elixir companies][elixir-companies] list.
 
 * [Culture Amp](https://cultureamp.com) ([Github](https://github.com/cultureamp))
   Culture Amp makes it easy to collect, understand and act on employee feedback.
-  Based in Melbourne with offices in San Francisco, New York, Palo Alto, and London.
+  Based in Melbourne with offices in San Francisco, New York, Palo Alto, and London. [Video about Elm at Culture Amp]().
```
Comment (nelsonic) [inline README.md:237]: Did you mean to leave the URL out of this Link?
Suggested link might be: 
+ Developer Happiness on the Front End with Elm: https://youtu.be/kuOCx0QeQ5c
+ Elm in Production: Surprises & Pain Points: https://youtu.be/LZj_1qVURL0
+ Elm at Scale: More Surprises, More Pain Points: https://youtu.be/uQjivmLan0E

File: README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ Inspired by Doomspork's [Elixir companies][elixir-companies] list.
 * [carwow](https://www.carwow.co.uk/) ([GitHub](https://github.com/carwow)) -
   Based in London and Munich, carwow is the most convenient, stress-free way to
   find and buy a new car, at a price you can be confident in, and we love Elm!
+* [Ford](https://www.ford.com/). [Talk about Elm at Ford](https://www.youtube.com/watch?v=5WVXCy1Q88o).
```
Comment (nelsonic) [inline README.md:18]: Great Elm Conf talk. Consider adding the title of the talk as metadata so people can see more when they hover over the link:
```suggestion
* [Ford](https://www.ford.com/). [Talk about Elm at Ford (Elm Conf)](https://www.youtube.com/watch?v=5WVXCy1Q88o "Facilitating Technology Change Through Cultural Change - Grant Maki").
```

**Review by nelsonic — APPROVED** [review]
@lucamug great additions. 🎉 
Made a suggestion and noted that the Culture Amp link is empty.

File: README.md (hunk 1 above)
Comment (lucamug) [reply README.md:237]: I was thinking of "Elm in Production: Surprises & Pain Points". Do you think it makes sense to add all of them?

Comment (nelsonic) [reply README.md:237]: Each of Kevin's videos cover a different stage in Culture Amp's experience of using Elm. You can include one of them and YT will recommend the rest _or_ include them all and let the viewer decide which one interests them most. 💭

File: README.md [hunk 3]
```diff
@@ -271,6 +272,8 @@ Inspired by Doomspork's [Elixir companies][elixir-companies] list.
   SaaS apps' usage and spend, along with powerful workflows to manage change.
   Headquarters in NYC, engineering is remote friendly, with 50% outside NYC.
 * [Stax](https://www.stax.io/) - Secure platform to AWS cloud management and monitoring. Melbourne, Australia.
+* [Humlio](https://www.humio.com/) - Humio is the creator of the Humio live observability platform that enables data aggregation, exploration, reporting and analysis from a range of sources. The purpose-built logging tool features innovative data storage and in-memory search/query engine technologies. [Post about Elm at Humlio](https://www.humio.com/whats-new/blog/why-we-chose-elm-for-humio-s-web-ui).
```
Comment (adamdicarlo) [inline README.md:275]: typo at start of bullet: "Humlio"

Comment (jfmengels) [reply README.md:275]: Humio is already in the list

File: README.md [hunk 4]
```diff
@@ -271,6 +272,8 @@ Inspired by Doomspork's [Elixir companies][elixir-companies] list.
...
+* [Thoughtbot](https://thoughtbot.com/) - Thoughtbot is your expert design and development partner. We bring digital products from validation to success and teach you how because we care. It’s time to move beyond simply building features and start designing the right product with the right strategy. [Posts about Elm at Thoughtbot](https://thoughtbot.com/blog/tags/elm).
```
Comment (jfmengels) [inline README.md:276]: Thoughtbot is also already in the list

**Pull request closed without merging**